CONTRIBUTING.md

```diff
@@ -375,7 +411,7 @@ You should now have a local instance of the site running at `http://localhost:50
 
 After certain kinds of changes are made, you need to build the project before serving it from a webserver will work
 
-* CSS: a build will run cssnext, combine custom media queries, and concat all source files into webcompat.dev.css. You'll need to re-build the CSS to see any changes, so it's recommended to use a watch task (see `make watch` or `grunt watch`).
+* CSS: a build will run cssnext, combine custom media queries, and concat all source files into webcompat.dev.css. You'll need to re-build the CSS to see any changes, so it's recommended to use a watch task (see `make watch` or `npm run watch`).
```

Comment:
We removed the Makefile in https://github.com/webcompat/webcompat.com/commit/adc48ef679c1009a987b58f2ab3dcbc75155171e, so we can remove the `make watch` command.